configure.ac

```diff
@@ -6,15 +6,15 @@ AC_PREREQ([2.60])
 define(_PKG_VERSION_MAJOR, 0)
 define(_PKG_VERSION_MINOR, 3)
 define(_PKG_VERSION_PATCH, 1)
-define(_PKG_VERSION_IS_RELEASE, false)
+define(_PKG_VERSION_IS_RELEASE, true)
 
 # The library version is based on libtool versioning of the ABI. The set of
 # rules for updating the version can be found here:
 # https://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
 # All changes in experimental modules are treated as if they don't affect the
 # interface and therefore only increase the revision.
 define(_LIB_VERSION_CURRENT, 2)
-define(_LIB_VERSION_REVISION, 1)
+define(_LIB_VERSION_REVISION, 2)
```

Comment:
Why increase the lib version?

Reply:
indeed.

Reply:
Maybe I'm missing something obvious, but this library version was never released. I'm not sure why we would treat the library version different to the package version in that regard. The lib version was incremented in [#1228](https://github.com/bitcoin-core/secp256k1/pull/1228/commits/28e63f7ea75af50af0bfde34309993f276bbc0ae) at the same time when the package version was updated to 0.3.1. We do this so that master has a different version than the previous release.